src/BL/TransactionManager.java

```diff
@@ -0,0 +1,64 @@
+package BL;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+public class TransactionManager {
+    // Определяет приватное поле dataRate, предназначенное для хранения экземпляра класса CurrencyExchangeHandler
+    private final CurrencyExchangeHandler dataRate;
+    // Определяет приватное поле dataHistory, предназначенное для хранения экземпляра класса TransactionHistoryHandler
+    private final TransactionHistoryHandler dataHistory;
+
+    // Конструктор класса
+    public TransactionManager(String rateFilePath, String historyFilePath ) {
```

Comment:
here should be the same approach as in CurrencyManager - no parameters and path from constant variable _FILE_PATH